src/registrar/tests/test_emails.py

```diff
@@ -112,9 +111,7 @@ def test_submission_confirmation_other_contacts_spacing(self):
         _, kwargs = self.mock_client.send_email.call_args
         body = kwargs["Content"]["Simple"]["Body"]["Text"]["Data"]
         self.assertIn("Other employees from your organization:", body)
-        # spacing should be right between adjacent elements
-        self.assertRegex(body, r"5556\n\nOther employees")
```

Comment:
the "5556" was the ending characters of the Submitter (last digits of their phone) and no longer needs to be tested for if there's space in between given we're removing submitter

Reply:
Figured out that user wasn't being passed in with Zander and will push up code that resolves this!